doc/howto/container_gpu_passthrough_with_docker.md

```diff
@@ -0,0 +1,155 @@
+        lxc config device add t1 gpu0 gpu gputype=physical id=nvidia.com/gpu=0
+
+Let's try to run a basic [MNIST](https://en.wikipedia.org/wiki/MNIST_database) inference job inside our LXD container.
+
+1. For that, create a `cloud-init` script that installs the Docker runtime, the [NVIDIA Container Toolkit](https://github.com/NVIDIA/nvidia-container-toolkit), and a script to run a test [TensorRT](https://github.com/NVIDIA/TensorRT) workload:
+
+        #cloud-config
+        package_update: true
+        packages:
+          - docker.io
+        write_files:
+          - path: /etc/docker/daemon.json
+            permissions: "0644"
+            owner: root:root
+            content: |
+            {
+                "runtimes": {
+                  "nvidia": {
+                    "args": [],
+                    "path": "nvidia-container-runtime"
+                  }
+                }
+            }
+          - path: /root/run_tensorrt.sh
+            permissions: "0755"
+            owner: root:root
+            content: |
+            #!/bin/bash
+            echo "OS release,Kernel version"
+            (. /etc/os-release; echo "${PRETTY_NAME}"; uname -r) | paste -s -d,
+            echo
+            nvidia-smi -q
+            echo
+            exec bash -o pipefail -c "
+            cd /workspace/tensorrt/samples
+            make -j4
+            cd /workspace/tensorrt/bin
+            ./sample_onnx_mnist
+            retstatus=\${PIPESTATUS[0]}
+            echo \"Test exited with status code: \${retstatus}\" >&2
+            exit \${retstatus}
+            "
+        runcmd:
+          - systemctl start docker
+          - systemctl enable docker
+          - usermod -aG docker root
+          - curl -fsSL https://nvidia.github.io/libnvidia-container/gpgkey | sudo gpg
+            --dearmor -o /usr/share/keyrings/nvidia-container-toolkit-keyring.gpg
+          - curl -s -L
+            https://nvidia.github.io/libnvidia-container/stable/deb/nvidia-container-toolkit.list
+            | sed 's#deb https://#deb
+            [signed-by=/usr/share/keyrings/nvidia-container-toolkit-keyring.gpg]
+            https://#g' | tee /etc/apt/sources.list.d/nvidia-container-toolkit.list
+          - sed -i -e '/experimental/ s/^#//g'
+            /etc/apt/sources.list.d/nvidia-container-toolkit.list
+          - apt-get update
+          - DEBIAN_FRONTEND=noninteractive apt-get install -y nvidia-container-toolkit
+          - nvidia-ctk runtime configure --runtime=docker
+            --config=/etc/docker/daemon.json
+          - systemctl restart docker
+
+1. Apply this `cloud-init` setup to your instance:
+
+        lxc config set t1 cloud-init.user-data - < cloud-init.yml
+
+1. Start the instance:
+
+        lxc start t1
+
+1. Wait for the `cloud-init` process to finish:
+
+        lxc exec t1 -- cloud-init status --wait
+
+1. Once cloud-init is finished, we can shell into the instance:
+
+        lxc shell t1
```

Comment:
```suggestion
1. Once cloud-init is finished, open a shell in the instance:

        lxc exec t1 -- /bin/bash
```

Reply:
(We don't use the `shell` alias in the docs - and also, there is no verb "to shell". :wink: )

Reply:
fixed